ibm_cloud_sdk_core/api_exception.py

```diff
@@ -13,17 +13,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from requests import Response
+from typing import Optional
+
 
 class ApiException(Exception):
-    """
-    Custom exception class for errors returned from APIs.
+    """Custom exception class for errors returned from operations.
 
-    :param int code: The HTTP status code returned.
-    :param str message: A message describing the error.
-    :param dict info: A dictionary of additional information about the error.
-    :param response http_response: response
+    Args:
+        code: HTTP status code of the error response.
+        message: The error response body. Defaults to None.
+        http_response: The HTTP response of the failed request. Defaults to None.
+
+    Attributes:
+        code (int): HTTP status code of the error response.
+        message (str): The error response body.
+        http_response (requests.Response): The HTTP response of the failed request.
+        global_transaction_id (str, optional): Globally unique id the service endpoint has given a transaction.
```

Comment:
I was a little surprised by the change in documentation style, but then discovered that this new style is exactly what our "CONTRIBUTING" doc says to use.  So nice job in making us adhere to our own style guide!

Reply:
👍